homeassistant/components/hue/strings.json

```diff
@@ -102,6 +102,25 @@
         }
       }
     },
+    "light": {
+      "hue_light": {
+        "state_attributes": {
+          "effect": {
+            "state": {
+              "none": "None",
+              "candle": "Candle",
+              "sparkle": "Sparkle",
+              "glisten": "Glisten",
+              "sunrise": "Sunrise",
+              "sunset": "Sunset",
+              "fire": "Fire",
+              "prism": "Prism",
+              "opal": "Opal"
```

Comment:
A couple of new effects have been added to the library. If we release a new lib version, we can make use of those here as well. Do that first ?

Reply:
If we bump the lib first, we can add it to a patch release, if we do this first, it has to wait for 2025.3